datalad/metadata/tests/test_search.py

```diff
@@ -358,3 +361,80 @@ def test_meta2autofield_dict():
             'extr1': {'prop1': 'value'}}),
         {'extr1.prop1': 'value'}
     )
+
+
+def test_external_indexer():
+    """ check that external indexer are called """
+    class MockedIndexer:
```

Comment:
shouldn't it subclass `MetadataIndexer`?

Reply:
It could be. Then we would not only test that externally available indexers are loaded, but also that the class-hierarchy is properly designed/implemented.

Reply:
yeap, that is the idea... in particular if abc and `@abstractmethod` are used

Reply:
Done